app/helpers/anchor/form_builder.rb

```diff
@@ -0,0 +1,122 @@
+module Anchor
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    delegate :concat, :content_tag, :render, to: :@template
+
+    def error_message_for(attribute, options = {})
+      render ErrorMessageComponent.new(
+        form_builder: self,
+        attribute:,
+        **options
+      )
+    end
+
+    def label(attribute, text = nil, options = {}, &)
+      render LabelComponent.new(
+        form_builder: self,
+        attribute:,
+        text:,
+        **options
+      ) do |component|
+        component.with_label_tag do
+          super attribute, text, options.merge(component.options), &
+        end
+      end
+    end
+
+    def text_field(attribute, options = {})
+      render TextFieldComponent.new(
+        form_builder: self,
+        attribute:
+      ) do |component|
+        component.with_input_tag do
+          super attribute, options.merge(component.options)
```

Comment:
Should this be the other way around, to allow overriding options?

```suggestion
          super attribute, component.options.merge(options)
```

Reply:
Good call @MatheusRich! Done.